flink-connectors/flink-connector-hive/src/test/java/org/apache/flink/table/catalog/hive/HiveTestUtils.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.table.catalog.hive;
+
+import org.apache.hadoop.hive.conf.HiveConf;
+import org.junit.rules.TemporaryFolder;
+
+import java.io.IOException;
+
+/**
+ * Test utils for Hive connector.
+ */
+public class HiveTestUtils {
+	private static final String HIVE_SITE_XML = "hive-site.xml";
+	private static final String HIVE_WAREHOUSE_URI_FORMAT = "jdbc:derby:;databaseName=%s;create=true";
+	private static final TemporaryFolder TEMPORARY_FOLDER = new TemporaryFolder();
+
+	private static String warehouseDir;
+	private static String warehouseUri;
+
+	/**
+	 * Create a GenericHiveMetastoreCatalog with an embedded Hive Metastore.
+	 */
+	public static GenericHiveMetastoreCatalog createGenericHiveMetastoreCatalog() throws IOException {
+		return new GenericHiveMetastoreCatalog("test", getHiveConf());
+	}
+
+	private static HiveConf getHiveConf() throws IOException {
+		ClassLoader classLoader = new HiveTestUtils().getClass().getClassLoader();
+		HiveConf.setHiveSiteLocation(classLoader.getResource(HIVE_SITE_XML));
+
+		TEMPORARY_FOLDER.create();
+		warehouseDir = TEMPORARY_FOLDER.newFolder().getAbsolutePath() + "/metastore_db";
+		warehouseUri = String.format(HIVE_WAREHOUSE_URI_FORMAT, warehouseDir);
+		HiveConf hiveConf = new HiveConf();
+		hiveConf.setBoolVar(HiveConf.ConfVars.METASTORE_SCHEMA_VERIFICATION, false);
```

Comment:
It seems some of these configs are already in the hive-site.xml

Reply:
good catch. I will keep those in hive-site.xml